src/crimson/net/Throttle.h

```diff
@@ -0,0 +1,36 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:nil -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include <core/condition-variable.hh>
+
+#include "common/ThrottleInterface.h"
+
+namespace ceph::net {
```

Comment:
this throttle seems generally useful outside of the messenger, maybe ceph::thread instead?